src/helpers/featureFlags.js

```diff
@@ -0,0 +1,16 @@
+/* To set a feature flag:
+
+In the browser dev tools console, type:
```

Comment:
we just need to: either hide the page, point to the existing shiny page or another page instead of using featureflag. The problem is it requires another command from user/dev